src/ngraph/frontend/onnx_import/op/add.cpp

```diff
@@ -0,0 +1,59 @@
+//*****************************************************************************
+// Copyright 2017-2019 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include "add.hpp"
+#include "default_opset.hpp"
```

Comment:
This `add.cpp` file was created to avoid include `default_opset.hpp` in `add.hpp` file,
(default_opset file contain `using namespace` statement).
It should be done for all ops with only `*.hpp` files, if we want to use `default_opset.hpp` idea.